### PR TITLE
AVX-17102: Use marketplace image to create controller and remove image variable

### PR DESCRIPTION
**Pull request by mlin-aviatrix**

### CODE REVIEW

File: aviatrix-controller-build/README.md [hunk 1]
```diff
@@ -53,10 +51,6 @@ module "aviatrix-controller-build" {
 
   The CIDR for the Google Subnetwork that will be created. Must be empty if **network** is set. Default value is "10.128.0.0/9".
```
Comment (zhiqiangjin) [inline aviatrix-controller-build/README.md:52]: No indent. Please keep the format consistent with other modules.

File: aviatrix-controller-build/README.md [hunk 2]
```diff
@@ -53,10 +51,6 @@ module "aviatrix-controller-build" {
...
 
-- **image**
-
-  The name or self_link of an existing Google compute image with the Aviatrix Controller Image. If not set, a Google Compute Image with the Aviatrix Controller image will be created.
```
Comment (zhiqiangjin) [inline aviatrix-controller-build/README.md:58]: Same here.

File: aviatrix-controller-build/README.md [hunk 3]
```diff
@@ -53,10 +51,6 @@ module "aviatrix-controller-build" {
...
-
 - **public_ip**
 
   Existing public IP Address to assign to the Aviatrix Controller. If not set, a Google Compute public IP address will be created.
```
Comment (zhiqiangjin) [inline aviatrix-controller-build/README.md:56]: Same.

**Review by zhiqiangjin — CHANGES_REQUESTED** [review]

**Review by shuoliugit — APPROVED** [review]

File: aviatrix-controller-build/README.md (hunk 1 above)
Comment (mlin-aviatrix) [reply aviatrix-controller-build/README.md]: I was using the same format as the [AWS module](https://github.com/AviatrixSystems/terraform-modules/blob/master/aviatrix-controller-build/README.md), which has an indent there to so the text is that same indent as the bullet mark.

**Pull request merged**